sessions/TypeScript-100/common-types-examples.ts

```diff
@@ -0,0 +1,150 @@
+const isFun: boolean = false;
+
+// there is no int or float, everything is number to JS/TS
+const pie: number = 3.14;
+
+// strings can use single or double quotes
+const fullName: string = "Bob Bobbington";
+
+// template string, using backticks
+const greet: string = `Salutations ${fullName}!`
+
+// symbol
+// note that both variables have what seems to be the same value
+const firstName = Symbol("name");
```

Comment:
```suggestion
const firstName = symbol("name");
```
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#number-string-boolean-symbol-and-object

```
❌ Don’t ever use the types Number, String, Boolean, Symbol, or Object These types refer to non-primitive boxed objects that are almost never used appropriately in JavaScript code.
```